.gitignore

```diff
@@ -28,3 +28,10 @@ ros_tcp/ros_communication_client.egg-info/**
 spot_rl_experiments/experiments/skill_test/episode_configs/*
 *.npy
 *.csv
+*.json
+*.pkl
+*.gz
+*.pt*.pkl*
```

Comment:
`*.pt*.pkl*` is a bit repetitive, we can change lines 32 and 34 to:
```
*.pkl*
*.pt
```

Reply:
Done